Esri/military-features-data
# Updated the APP6-B stlyx file

Fixes issue #291 
Actions to verify: 
1. Open sylx file in tksqlite (please let me know if you need this software)
2. Use the search commands to verify that no records are returned. 

Updated the stylx file to remove spaces from all tags where a space was before and after the semi colon.

Commands used:

UPDATE COMMANDS
UPDATE ITEMS SET TAGS = REPLACE(TAGS, ' ;', ';')

UPDATE ITEMS SET TAGS = REPLACE(TAGS, ' ;', ';')


Search Commands

select * from ITEMS where TAGS like  '%; %'

select * from ITEMS where TAGS like  '% ;%'



## Description


## Related Issue





## Motivation and Context


## How Has This Been Tested?




## Screenshots (if appropriate):

## Types of changes


## Checklist:
